ksql-engine/src/main/java/io/confluent/ksql/planner/plan/JoinNode.java

```diff
@@ -49,6 +51,8 @@
     INNER, LEFT, OUTER
   }
 
+  private static final String JOIN_OP_NAME = "join";
```

Comment:
Same comment as above: the name `JOIN_OP_NAME` is somewhat confusing in light of `joinOpName()` below.